Gate_system/Models/ReportItem.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Gate_system.Models
+{
+    public class ReportItem
+    {
+
+        private int id;
```

Comment:
Microsofto convention'as yra privacias variables vadinti su bruksneliu ir is mazosios raides, pvz _firstName, _lastName ir t.t.